### PR TITLE
[IB] remove engine from the settings

### PR DESCRIPTION
**Pull request by lanking520**
## Description ##

Remove engine from the settings

### CODE REVIEW

File: tests/integration/benchmark/nightly/g5-12xl.txt [hunk 1]
```diff
@@ -5,7 +5,6 @@ ENGINE={vllm,lmi-dist}
 [container]
 deepjavalibrary/djl-serving:lmi-nightly
 [serving_properties]
-engine=Python
```
Comment (frankfliu) [inline tests/integration/benchmark/nightly/g5-12xl.txt:8]: Why removing it?

It doesn't hurt and make it easy to distinguish from future rust engine

Comment (lanking520) [reply tests/integration/benchmark/nightly/g5-12xl.txt:8]: this is wrong today. For LMI-Dist, we will need MPI engine

Comment (frankfliu) [reply tests/integration/benchmark/nightly/g5-12xl.txt:8]: We shouold automatically add `option.mpi_model=true`, if not then it's a bug

Comment (lanking520) [reply tests/integration/benchmark/nightly/g5-12xl.txt:8]: it is added

**Pull request closed without merging**